modules/custom/activity_creator/activity.page.inc

```diff
@@ -84,7 +84,7 @@ function template_preprocess_activity(array &$variables) {
   // 'compact_notification' view mode might not be configured, so remember to
   // always check the theme setting first.
   $storage = \Drupal::entityTypeManager()->getStorage('profile');
-  if ($storage !== NULL) {
+  if ($storage !== NULL && $account !== NULL) {
```

Comment:
Hi @navneet0693 
This causes an PHPStan issue in main, could you perhaps check that out?

Reply:
@ronaldtebrake Sure, I will open PR for fixing it, I spotted it as well on https://github.com/goalgorilla/open_social/pull/3060 and https://github.com/goalgorilla/open_social/pull/2877